src/components/Loading/index.tsx

```diff
@@ -0,0 +1,17 @@
+import { FC } from 'react';
+import Spinner from './components/Spinner';
+
+type LoadingViewType = 'spinner';
+
+interface LoadingViewProps {
+  type?: LoadingViewType;
+}
+
+const LoadingView: FC<LoadingViewProps> = ({ type = 'spinner' }) => {
```

Comment:
```suggestion
const LoadingView = ({ type = 'spinner' }: LoadingViewProps) => {}
```

React.FC는 지양하는 게 좋아서 이렇게 작성하면 어떨까요?
[요기](https://yceffort.kr/2022/03/dont-use-react-fc) 참고하면 좋을 것 같아요!

Reply:
우선 토픽 공유주셔서 감사해요 ! 알고 있던 토픽인데 현재는 해당하지 않는 이슈입니다.
https://github.com/DefinitelyTyped/DefinitelyTyped/pull/56210

컨벤션 영역이라고 생각되어서 차주 회의에 얘기해보면 좋을것 같습니다.

Reply:
요약하면 React18부터는 children을 암묵적으로 가지지 않습니다.